sdk/src/main/scala/com/akkaserverless/javasdk/testkit/Result.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.javasdk.testkit
+
+import scala.reflect.ClassTag
+import java.util.NoSuchElementException
+
+private[testkit] class Result[Reply](val reply: Reply, events: java.util.List[Any]) {
```

Comment:
`Result` is public API so I think this should be written in Java, or be an interface in Java with a Scala impl.